app/sentry.ts

```diff
@@ -10,8 +11,10 @@ if (!config.sentry.dsn.includes('dummy')) {
             new Dedupe(),
             new CaptureConsole({
                 levels: ['error']
-            })
+            }),
```

Comment:
Codacy found an issue: [Expected indentation of 6 spaces but found 12.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5948471)